src/main/java/DukeException.java

```diff
@@ -0,0 +1,5 @@
+public class DukeException extends Exception{
```

Comment:
I think you should add a space after Exception. I noticed that this is a recurring practice for your classes. Adding a space between the class declaration and the { will help to add readability to your code. I suggest you read the part on "White space within a statement" in the Java coding standard 😄 .